components/CredentialDetailsTree.vue

```diff
@@ -33,8 +33,13 @@ export default {
       const value = toRaw(obj[key] || obj);
       const node = {id: crypto.randomUUID(), label: key, children: []};
       if(Array.isArray(value)) {
-        value.forEach(subValue => {
-          createNodeList(subValue, value[subValue] || {}, node.children);
+        // if value is an array of objects, make each key its index
+        value.forEach((subValue, index) => {
+          if(typeof value[0] === 'object') {
```

Comment:
This is confusing ... since it will always evaluate the same way. Is this the right check? If so, it should be outside of the `forEach` loop.

Reply:
Updated the function here to move the boolean check outside of the loop: b527def